vendor-bin/php-codesniffer/composer.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
**New files**

Everything in `vendor-bin`, as well as `phpcs.xml`, `phpstan.neon`, `psalm.xml` added for static code analysis